4.sc_Morphology_Neural_Network_MLP_Model/0.hyperparameter_optimization/notebooks/Hyperparameter_Optimization_model_regression_call.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+#SBATCH --nodes=1
+#SBATCH --ntasks=16
+#SBATCH --mem=300G
+#SBATCH --partition=amem
+#SBATCH --qos=mem
+#SBATCH --time=24:00:00
+#SBATCH --output=sample-%j.out
+
+# module load anaconda
+
+# conda activate Interstellar
+
+cell_types=( SHSY5Y )
+control_names=( DMSO_0.100_DMSO_0.025 )
+treatment_names=( LPS_100.000_DMSO_0.025 Thapsigargin_10.000_DMSO_0.025 LPS_10.000_DMSO_0.025 LPS_1.000_DMSO_0.025 LPS_0.100_DMSO_0.025 LPS_0.010_DMSO_0.025 Thapsigargin_1.000_DMSO_0.025 )
+model_names=( DMSO_0.025_vs_LPS_100 DMSO_0.025_vs_Thapsigargin_10 DMSO_0.025_vs_LPS_10 DMSO_0.025_vs_LPS_1 DMSO_0.025_vs_LPS_0.1 DMSO_0.025_vs_LPS_0.01 DMSO_0.025_vs_Thapsigargin_1 )
```

Comment:
Consider quoting the strings found in these bash arrays to avoid issues where spaces or other special characters are treated differently (and sometimes difficult to troubleshoot).

Reply:
Good point, I added these in